lib/keyring_liberator.rb

```diff
@@ -21,12 +21,16 @@ def self.get_keyring(path)
     end
 
     def self.get_keyring_named(name)
-      get_all_keyrings.find { |k| k.name == name }
+      get_all_keyrings.find { |k| k.name == name && k.path == Dir.getwd }
     end
 
     def self.save_keyring(keyring)
       keys_dir.mkpath
 
+      if get_keyring_named(keyring)
+        puts "About to create a duplicate keyring file for project #{keyring.name.green}"
+      end
```

Comment:
Gotcha.  So you're talking about this section, and adding a prompt to continue or back out?  I'll toss that in.